src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp

```diff
@@ -189,25 +189,47 @@ bool FixConservatives::operator()(
       //   converge. We handle this case explicitly because we found that
       //   setting a lower bound of 1.-1.e-12 leads to floating point
       //   exceptions.
+      // - In regions where the solution is just above atmosphere we sometimes
+      //   obtain an upper bound on the Lorentz factor somewhere around ~1e5,
+      //   while the actual Lorentz factor is only 1+1e-6. This leads to
+      //   situations where the solver must perform many (over 50) iterations to
+      //   converge. A simple way of avoiding this is to check that
+      //   10 * W_{lower_bound} brackets the root and then use that as the upper
```

Comment:
- [nitpick] technically, [W_{lower_bound}, 10 * W_{lower_bound}] brackets the root... how about rephrasing to say "... to check that [W_{lower_bound}, 10 * W_{lower_bound}] brackets the root and then use 10 * W_{lower_bound} as the upper bound" ?
- why 10? I assume it's largely arbitrary and works? Can you state that in the text, to guide future users if the value needs to be revisited?